### ISSUE TITLE
ability to escalate pagerduty page to fe team

### ISSUE THREAD

**Issue by cgardens** [airbytehq/airbyte#11673]
## Tell us about the problem you're trying to solve
we want to be able to escalate any on call page to someone on the frontend. figure out how to make it easy for a platform on call engineer to escalate any given issue to the frontend rotation.

the idea is that the frontend rotation will never be a first line page, but can always be escalated to.

**Comment by supertopher** [comment]
Topher (or other human) 

you will know you are done when:

- [] there is a pager duty group for the FE team
- [] there is an escalation path ending with an Charles 
- [] there is an path to reach this team that does not require a pager duty account 
- [] Promulgate this escalation path or create a new issue to track that 

![](https://phoenixeliquid.co.uk/wp-content/uploads/2016/06/The-Big-Red-Panic-Button.png)

**Comment by pmossman** [comment]
@supertopher does this issue require any work? @timroes it looks like you're the permanent OC for https://airbyte.pagerduty.com/schedules#P65VJEK, I imagine we can simply escalate an issue to that schedule if we need frontend visibility?

**Comment by cgardens** [comment]
I think we were having fantasies about the following:
page comes in and has options: 1) resolve 2) acknowledge 3) escalate _4) escalate to frontend_

If indeed this is pie in the sky, I think just some way to redirect to the frontend rotation that is documented is a reasonable second best.

**Comment by timroes** [comment]
@pmossman Yeah, tough life, isn't it :wink: As soon as the rest of the team is a bit more onboarded we plan to add more FE people to that rotation.

In discussion with Peter and Topher yesterday, my suggested "escalation policy" (for now, until we have better timezone coverage with the rest of the team) would be:

* Is this clearly a FE issue?
  * no: try to involve either platform team OC first to ensure it's a FE issue (since the FE oncall rotation is very small atm)
   * yes: (or figured with platform it IS a FE issue) - then:
* Ask in `#dev-frontend` for help
* You're not getting a timely response and it is an urgent issue to solve i.e. can't wait 24h?
  * If yes, then escalate using Pagerduty: Create an issue in the "FE Escalation Policy" if there is no Pagerduty incident yet, or escalate to the "Frontend OC Urgent" within the Platform OC Escalation Policy if it already exists in Pagerduty.
